salt/modules/cmdmod.py

```diff
@@ -602,7 +603,7 @@ def _get_stripped(cmd):
         else:
             # On Windows set the codepage to US English.
             if python_shell:
```

Comment:
```suggestion
            if python_shell and chcp is not None:
                _run(cmd="chcp {0} > nul".format(chcp),
                        cwd=cwd,
                        stdin=stdin,
                        stdout=stdout,
                        stderr=stder,
                        output_encoding=output_encoding,
                        output_loglevel=output_loglevel=,
                        log_callback=log_callback,
                        runas=runas,
                        group=group,
                        shell=shell,
                       python_shell=python_shell,
                       env=env,
                       clean_env=clean_env,
                       prepend_path=prepend_path,
                       rstrip=rstrip,
                       template=template,
                       umask=umask,
                       timeout=timeout,
                       with_communicate=with_communicate,
                       reset_system_locale=reset_system_locale,
                       ignore_retcode=ignore_retcode,
                       saltenv=saltenv,
                       pillarenv=pillarenv,
                       pillar_override=pillar_override,
                       use_vt=use_vt,
                       password=password,
                       bg=bg,
                       encoded_cmd=encoded_cmd,
                       success_retcodes=success_retcodes,
                       chcp=None,
                       **kwargs)
```

Reply:
Why uses 437? Python is utf-8 so why not just use 65001 by default? Also the code page change does not take effect until the second call. So we can just call _run with just the code page change so it takes effect right away!